### ISSUE TITLE
Doesn't seem to work with Postgres

### ISSUE THREAD

**Issue by will-in-wi** [vapid/vapid#149]
**Describe the bug**
When trying to set up Vapid with Postgres, I get the following error:

```
GET /dashboard/ 500 1742 - 5.372 ms [-]
SequelizeDatabaseError: operator does not exist: json <> unknown
    at Query.formatError (/home/chofstillwater/webapps/marketing_site/marketing-site/node_modules/sequelize/lib/dialects/postgres/query.js:354:16)
    at query.catch.err (/home/chofstillwater/webapps/marketing_site/marketing-site/node_modules/sequelize/lib/dialects/postgres/query.js:71:18)
    at tryCatcher (/home/chofstillwater/webapps/marketing_site/marketing-site/node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (/home/chofstillwater/webapps/marketing_site/marketing-site/node_modules/bluebird/js/release/promise.js:517:31)
    at Promise._settlePromise (/home/chofstillwater/webapps/marketing_site/marketing-site/node_modules/bluebird/js/release/promise.js:574:18)
    at Promise._settlePromise0 (/home/chofstillwater/webapps/marketing_site/marketing-site/node_modules/bluebird/js/release/promise.js:619:10)
    at Promise._settlePromises (/home/chofstillwater/webapps/marketing_site/marketing-site/node_modules/bluebird/js/release/promise.js:695:18)
    at _drainQueueStep (/home/chofstillwater/webapps/marketing_site/marketing-site/node_modules/bluebird/js/release/async.js:138:12)
    at _drainQueue (/home/chofstillwater/webapps/marketing_site/marketing-site/node_modules/bluebird/js/release/async.js:131:9)
    at Async._drainQueues (/home/chofstillwater/webapps/marketing_site/marketing-site/node_modules/bluebird/js/release/async.js:147:5)
    at Immediate.Async.drainQueues [as _onImmediate] (/home/chofstillwater/webapps/marketing_site/marketing-site/node_modules/bluebird/js/release/async.js:17:14)
    at runCallback (timers.js:705:18)
    at tryOnImmediate (timers.js:676:5)
    at processImmediate (timers.js:658:5)
```

I'm running it with `NODE_ENV=production SECRET_KEY=mysecretkey yarn exec vapid start .`

My `package.json` looks like:
```json
{
  "name": "mysite",
  "vapid": {
    "port": 3001,
    "database": {
      "dialect": "postgres",
      "database": "dbname",
      "username": "uname",
      "password": "password"
    }
  },
  "scripts": {
    "start": "vapid start ."
  },
  "dependencies": {
    "@vapid/cli": "^0.8.5",
    "pg": "^7.12.1",
    "yarn": "^1.17.3"
  }
}
```

**To Reproduce**
1. Set up new vapid site with package.json as defined
2. Start it with command as specified.
3. Try to load dashboard

**Expected behavior**
I expected it to work.

**Your Environment**
Include relevant details, so others can replicate the issue.
 - OS: Linux
 - Browser: Firefox
 - Node Version: v10.16.0
 - Vapid Version: 0.8.5
 - Postgres Version: 9.4.23

**Comment by will-in-wi** [comment]
On an unrelated note, but in case others are searching and find this, MySQL < 5.7 doesn't work either, because of the lack of a JSON column. That's probably to be expected.

**Comment by srobbin** [comment]
@will-in-wi Thanks for letting me know. This _should_ be fixed now in v0.8.6.
